### ISSUE TITLE
Content shift on all browsers (Windows/PC platform only)

### ISSUE THREAD

**Issue by wachilt** [andreknieriem/simplelightbox#330]
Strange one this…

SimpleLighbox is working perfectly and expected on Macbook (tested in Safari, Firefox, Chrome and Edge).

However, when I test same site on Surface PC (Firefox, Chrome, Edge and Opera) I am experiencing content shft (to the left) when opening the same SimpleLighbox enabled images.

Looks like it could potentially be connected with right-hand side scrollbars not auto-hiding on the PC platform as they do in macOS, though I could also be wrong.

**Comment by Mtillmann** [comment]
Hi could you provide some screen capture of the issue? Do you experience similar issues with other sites? does the issue exist with the simplelightbox demo site?

**Comment by wachilt** [comment]
HI Martin,

Apologies… I should have checked the simplelightbox demo page first (which works perfectly on PC). Your suggestion did point me in the right direction though. The site I am currently working on has a fixed header and footer. It looks like these specific selectors are clashing with the CSS of simplelightbox.

**Comment by wachilt** [comment]
Thanks Andre,

I went through my site's core CSS (tiny, only around 30kb worth) and temporarily removed anything I thought may cause the issue. However, I was still seeing content shift on PC browsers only. I then went through simple-lightbox.css and did the same in an effort to trace/debug the issue. Found it.

`.sl-scrollbar-measure {
  position: absolute;
  top: -9999px;
  width: 50px;
  height: 50px;
  overflow: scroll;
}`

If I change overflow value to either `hidden` or `auto` the issue vanishes while maintaining expected lightbox functionality and behaviour.

For the record, I don't suggest this is an issue with simple-lightbox.css. I confirm/concur that your the simplelightbox demo site works perfectly when tested on same PC hardware. I think it's more to do with a property/value within one (or a combination) of my own selectors that, as yet, I have not been able to trace.
